src/libraries/Common/src/System/Net/Security/Unix/SafeFreeSslCredentials.cs

```diff
@@ -15,8 +16,8 @@ namespace System.Net.Security
 {
     internal sealed class SafeFreeSslCredentials : SafeFreeCredentials
     {
-        private SafeX509Handle _certHandle;
-        private SafeEvpPKeyHandle _certKeyHandle;
+        private SafeX509Handle _certHandle = null!;
```

Comment:
Both of these fields can be `null` if this object was created with `certificate == null`. Shouldn't these be nullable fields?

Reply:
Good catch, thanks.